src/vcpkg/commands.dependinfo.cpp

```diff
@@ -193,19 +193,17 @@ namespace vcpkg::Commands::DependInfo
             for (const auto& package : depend_info)
             {
                 const std::string name = package.package;
-                nodes.append(Strings::format("<Node Id=\"%s\" />", name));
+                fmt::format_to(std::back_inserter(nodes), "<Node Id=\"{}\" />", name);
```

Comment:
[no change requested] This should really use XmlSerializer...

Reply:
Yeah :(